rosdep/base.yaml

```diff
@@ -932,6 +933,7 @@ gstreamer0.10-plugins-ugly:
     wily: [gstreamer0.10-plugins-ugly]
 gstreamer0.10-pocketsphinx:
   fedora: [pocketsphinx-plugin]
+  gentoo: [app-accessibility/pocketsphinx]
```

Comment:
I only added the gentoo packages names. ROS still use old packages like `gstreamer0.10` or `QT4` and they have to decide to remove them (from ALL distros then) and to block users to depend on outdated packages.